src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -275,6 +276,10 @@ def require_package
 
   private
 
+  def set_tasks
+    @tasks = User.current.tasks if User.current && !User.current.is_nobody?
```

Comment:
Can a user be nil? I think we always have a user object so you can get rid of the `User.current` nil check, I think.